src/screens/ProfileScreen.tsx

```diff
@@ -23,6 +23,22 @@ const copyToClipboard = async (content: string) => {
   });
 };
 
+/**
+ * Return a string in the format of "X hrs Y min" or "Y min" depending on the
+ * the number of minutes passed.
+ * @param numMinutes Number of in minutes
+ * @returns string in the format of "X hrs Y min" or "Y min"
+ */
+const toHoursAndMinutes = (numMinutes: number) => {
```

Comment:
This should be tested and moved to utils folder.

Reply:
Moved and updated!